src/ibc_api/utils.py

```diff
@@ -1,18 +1,21 @@
 """API to fetch IBC data from EBRAINS via Human Data Gateway using siibra. 
 """
 
-import siibra
-from siibra.retrieval.repositories import EbrainsHdgConnector
-from siibra.retrieval.requests import EbrainsRequest, SiibraHttpRequestError
+# %$
+import json
 import os
-from tqdm import tqdm
+from datetime import datetime
+
 import nibabel
-from siibra.retrieval.cache import CACHE
+import numpy as np
```

Comment:
Did you use numpy anywhere? I couldn't find any instances from a quick look.

If you do indeed use it then it should be added to the `dependencies` in `pyproject.toml`, otherwise the import should be removed.